### PR TITLE
Await exec()

### PR DESCRIPTION
**Pull request by ericsciple**

### CODE REVIEW

File: kubectl.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /// <reference path="typings/globals/node/index.d.ts" />
 "use strict"
 
-import tl = require('vsts-task-lib');
+import tl = require('vsts-task-lib/task');
```
Comment (ericsciple) [inline kubectl.ts]: this was preventing the mock from getting called (real impl was getting called during test)

File: kubectl.ts [hunk 2]
```diff
@@ -28,11 +28,9 @@ async function run() {
         tl.debug("DEBUG:  " + kubectlbinary + " apply -f " + yamlfile + " --kubeconfig ./config")
 
         let kubectl: ToolRunner = tl.tool(kubectlbinary).arg('apply').arg('-f').arg(yamlfile).arg('--kubeconfig').arg('./config');
-        let result =  kubectl.execSync();
-        tl.debug("STDOUT: " + result.stdout);
-        tl.debug("STDERR: " + result.stdout);
+        await kubectl.exec();
```
Comment (ericsciple) [inline kubectl.ts]: default is to throw when exit code is not zero, so i didnt bother capturing the return code. exec only returns return code and doesnt return an object stdout/stderr like execSync.

if i recall correctly, by default stdout/stderr are written to the underlying process.stdout/stderr. if that is not desired, then there is an option to tool runner to make it silent.

File: kubectl.ts [hunk 3]
```diff
@@ -28,11 +28,9 @@ async function run() {
...
 
-        tl.setResult(tl.TaskResult.Succeeded, "kubectl works.")
+        tl.setResult(tl.TaskResult.Succeeded, "kubectl works.");
```
Comment (ericsciple) [inline kubectl.ts]: this semicolon wasn't necessary, i just happened to notice and adding it appeared to be consistent with the style present in the file :)

File: kubectl.ts (hunk 1 above)
Comment (ericsciple) [inline kubectl.ts]: this is a place i'll look at improving documentation in the task lib

**Comment by ericsciple** [comment]
fyi i used the versions you mentioned. this worked for me to build/run the tests:
```
npm install
npm install typings@2.1.0  --global-style
npm install typescript@2.1.5 --global-style
npm install tfx-cli@v0.3.45  --global-style
npm test
```

**Comment by ericsciple** [comment]
...but adding to devDependencies in the package.json would work too

**Comment by TsuyoshiUshio** [comment]
You are so kind! Really appreciate this! I'll add your note on my README. :)

File: kubectl.ts (hunk 1 above)
Comment (TsuyoshiUshio) [inline kubectl.ts]: This is the one! Thanks!

File: kubectl.ts (hunk 2 above)
Comment (TsuyoshiUshio) [inline kubectl.ts]: Sure.

File: kubectl.ts (hunk 3 above)
Comment (TsuyoshiUshio) [inline kubectl.ts]: Thanks!

**Pull request merged**